### DISCUSSION TITLE
Text import does not get automatically detected by Intellisense

### DISCUSSION THREAD

**Discussion by grahampcharles** [mantinedev/mantine#5787] [Q&A]
I've gotten used to the fact that `<Text>` will not be automatically detected by Intellisense, and I have to add the import manually:

`import { Text } from '@mantine/core';`

I'm just curious, though... why is that? Is that not somehow fixable? I would put it down to a name collision, but in those cases, Intellisense offers both imports (plus I'm not aware that I have other `<Text>` components in my project.

It's not a huge deal, just a matter of curiosity, mainly.

**Comment by rtivital** [comment]
It is not recognized because there is native js Text constructor. It cannot be fixed.
